### PR TITLE
fixed PUT /jobs/:job_id endpoint resolves #533

### LINKED ISSUE #533: Unable to update job.workers
```
$ grep workers some-job.json
    "workers": 2,
    "recycle_workers": 1000,

$ http PUT gen1:5678/jobs/12977f33-7688-4d3e-bbb8-58e7db3fc9d4 < some-job.json

$ http gen1:5678/jobs/12977f33-7688-4d3e-bbb8-58e7db3fc9d4 | jq '._updated, .workers'
"2017-09-08T23:33:24.962Z"
1
```

### PR DESCRIPTION
**Pull request by jsnoble**
needs to be in conjunction with teraslice client changes

### CODE REVIEW

File: integration-tests/spec/setup.js [hunk 1]
```diff
@@ -176,12 +176,13 @@ describe('teraslice', function() {
...
-    require('./cases/validation/job')();
+
+    require('./cases/cluster/api')();
+   // require('./cases/cluster/job-state')();
+    //require('./cases/data/id-reader')();
+    //require('./cases/data/elasticsearch-bulk')();
+    //require('./cases/data/reindex')();
+    //require('./cases/cluster/worker-allocation')();
+    //require('./cases/cluster/state')();
+    //require('./cases/validation/job')();
```
Comment (godber) [inline integration-tests/spec/setup.js]: Please uncomment these.

File: integration-tests/spec/setup.js [hunk 2]
```diff
@@ -176,12 +176,13 @@ describe('teraslice', function() {
...
-    
-    require('./cases/cluster/job-state')();
-    require('./cases/data/id-reader')();
-    require('./cases/data/elasticsearch-bulk')();
-    require('./cases/data/reindex')();
-    require('./cases/cluster/worker-allocation')();
-    require('./cases/cluster/state')();
-    require('./cases/validation/job')();
+
+    require('./cases/cluster/api')();
```
Comment (erik-stephens) [inline integration-tests/spec/setup.js]: Can use `fit()` or `fdescribe()` to run specific cases.

**Comment by godber** [comment]
This is the corresponding teraslice-client-js PR: https://github.com/terascope/teraslice-client-js/pull/7

**Comment by godber** [comment]
Passes with recently merge client changes: https://github.com/terascope/teraslice/issues/533

screen shot 2017-09-13 at 1 26 34 pm

**Pull request merged**